app/views/ingredients/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<% @ingredients.each do |ingredient| %>
+  <section id="ingredient-<%= ingredient.id %>">
+    <h1><%= ingredient.name %></h1>
```

Comment:
Nice job here. Just be mindful of how we want it to display:
```
As a visitor,
When I visit '/ingredients'
I see a list of all the ingredients including their name and cost
(e.g. "Ground Beef: 2"
      "Salt: 4")
```
So you could write it like:
```ruby
<p><%= ingredient.name %>: <%= ingredient.cost %></p>
```